Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -250,7 +250,18 @@ private extension MoyaProvider {
             let result = convertResponseToResult(response, request: request, data: data, error: error)
             // Inform all plugins about the response
             plugins.forEach { $0.didReceive(result, target: target) }
-            progressCompletion?(ProgressResponse(response: result.value))
+            if let progressCompletion = progressCompletion {
```

Comment:
Not a Nimble expert, so I do not find any sequence `containNil` predicate ( I try to call `contain` or `containElementSatisfying`)

I keep all progress objects into this array
```swift
var progressObjects: [Progress?] = []
```
then I test like this
```swift
 expect(progressObjects.filter{$0 != nil}.count) == 5
```
or if I test only the last
```swift
expect(progressObjects.last).toNot(beNil())
```